app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cp, filename, copy_filename = command.split()
+    if cp != "cp" or (
+            not filename.endswith(".txt")
+            or not copy_filename.endswith(".txt")
+    ):
```

Comment:
What about it?)

![image](https://user-images.githubusercontent.com/104218550/220344853-bc016e34-722d-40ac-934c-809712f91fb7.png)

replace your check of the file formats with this check. And remove redundant print and return, put the logic into one condition